### PR TITLE
Moved the accounts to dry_run=no

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [ ] enhancement
- [x] documentation
- [ ] dependencies

### CODE REVIEW

**Comment by athiruma** [comment]
@halbfin ptal 👋

**Review by ebattat — APPROVED** [review]
/LGTM

File: jenkins/tenant/aws/ecoeng_01/README.md [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+**POLICIES_IN_ACTION
+** = '["unattached_volume", "ip_unattached", "zombie_snapshots", "unused_nat_gateway", "s3_inactive", "empty_roles"]'
 
 Accounts:
+
 - **industry-partners**: "hhalbfin@redhat.com",
+- **certification-pipeline**: "matt.dorn@redhat.com, mhillsma@redhat.com, babak@redhat.com, hhalbfin@redhat.com",
+- **ecoengverticals-qe**: "augol@redhat.com, hhalbfin@redhat.com",
+- **emerge-partner**: "jsalomon@redhat.com, ltomasbo@redhat.com, hhalbfin@redhat.com",
+- **telco5g-ci**: "yjoseph@redhat.com, hhalbfin@redhat.com",
```
Comment (halbfin) [inline jenkins/tenant/aws/ecoeng_01/README.md:12]: add sshnaidm@redhat.com to *telco5g-ci" list

**Review by halbfin — APPROVED** [review]

**Review by ebattat — APPROVED** [review]
/lgtm

**Pull request merged**